tests/t002-ceed.c

```diff
@@ -6,15 +6,17 @@
 
 int main(int argc, char **argv) {
   Ceed ceed;
+  const char *backend = argv[1];
```

Comment:
Why these changes? This would be the only test with this change.

Reply:
For backends with arguments, we should match only on the resource prefix (i.e. `/cpu/occa/<args>`). If we standardize on the arguments like the other comment, we can check for a delimiter like `?` instead

Reply:
The intent of this test is to verify that the backend uniquely identified by the user matches the unique registered backend resource name of the backend that is that is being used.